internal/remotestorage/remotestorage.go

```diff
@@ -0,0 +1,65 @@
+package remotestorage
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+)
+
+// RemoteStorage defines functions to interact with application storage service
+type RemoteStorage interface {
+	Upload(url, fileName, formType string) ([]byte, error)
+}
+
+// Storage implements functions for RemoteStorage interface
+type Storage struct {
+	HTTPClient *http.Client
+}
+
+// NewRemoteStorage returns an implementation for RemoteStorage
+func NewRemoteStorage() RemoteStorage {
```

Comment:
Stutter. Since the package is already called `remotestorage`, calling the method `New()` would be enough, thus ending up `remotestorage.New()` from the caller's perspective :)

Reply:
Updating :)